dsl/src/main/scala/org/http4s/dsl/Http4sDsl.scala

```diff
@@ -44,6 +43,8 @@ trait Http4sDsl2[F[_], G[_]] extends RequestDsl with Statuses with Responses[F,
     */
   val → : impl.->.type = impl.->
 
+  val PathVar: impl.PathVar.type = impl.PathVar
```

Comment:
To fix bincompat this needs to be a `def` or a `lazy val`.

```suggestion
  lazy val PathVar: impl.PathVar.type = impl.PathVar
```

Reply:
Thanks for the help, this did solve the mima isssue locally. :3